firmware/trezor.h

```diff
@@ -31,6 +31,10 @@
 #define DEBUG_LINK 0
 #endif
 
+#ifndef DEBUG_RNG
+#define DEBUG_RNG 0
```

Comment:
That is true, unless `-Wundef` is used and that is exactly what we have in the `Makefile`.